packages/core/src/Table/TableCellHead/Action.js

```diff
@@ -15,12 +17,16 @@ export const Action = ({ onClick, children }) => (
                 justify-content: center;
                 flex-shrink: 0;
             }
-            span:hover {
+            button:hover {
```

Comment:
As the focused button outline has been removed, we need to highlight the fact that a button is focused with the keyboard using the background (otherwise keyboard users won't be able to tell where focus is):

```suggestion
            button:hover, button:focus-visible {
```